### ISSUE TITLE
Having errors on downloadng some boards

### ISSUE THREAD

**Issue by hrzboy** [magicdawn/node-huaban-downloader#1]
```
D:\hb>huaban https://huaban.com/boards/30870249/
SyntaxError: Unexpected end of input
    at Object.parse (native)
    at getBoard (C:\Documents and Settings\Administrator\Application Data\npm\no
de_modules\node-huaban\lib\index.js:42:15)
    at getPagePins (C:\Documents and Settings\Administrator\Application Data\npm
\node_modules\node-huaban\lib\index.js:51:17)
    at C:\Documents and Settings\Administrator\Application Data\npm\node_modules
\node-huaban\lib\index.js:76:19
    at next (native)
    at onFulfilled (C:\Documents and Settings\Administrator\Application Data\npm
\node_modules\node-huaban\node_modules\co\index.js:65:19)
    at tryCatcher (C:\Documents and Settings\Administrator\Application Data\npm\
node_modules\node-huaban\node_modules\bluebird\js\release\util.js:16:23)
    at Promise._settlePromiseFromHandler (C:\Documents and Settings\Administrato
r\Application Data\npm\node_modules\node-huaban\node_modules\bluebird\js\release
\promise.js:512:31)
    at Promise._settlePromise (C:\Documents and Settings\Administrator\Applicati
on Data\npm\node_modules\node-huaban\node_modules\bluebird\js\release\promise.js
:569:18)
    at Promise._settlePromise0 (C:\Documents and Settings\Administrator\Applicat
ion Data\npm\node_modules\node-huaban\node_modules\bluebird\js\release\promise.j
s:614:10)
    at Promise._settlePromises (C:\Documents and Settings\Administrator\Applicat
ion Data\npm\node_modules\node-huaban\node_modules\bluebird\js\release\promise.j
s:693:18)
    at Async._drainQueue (C:\Documents and Settings\Administrator\Application Da
ta\npm\node_modules\node-huaban\node_modules\bluebird\js\release\async.js:133:16
)
    at Async._drainQueues (C:\Documents and Settings\Administrator\Application D
ata\npm\node_modules\node-huaban\node_modules\bluebird\js\release\async.js:143:1
0)
    at Immediate.Async.drainQueues [as _onImmediate] (C:\Documents and Settings\
Administrator\Application Data\npm\node_modules\node-huaban\node_modules\bluebir
d\js\release\async.js:17:14)
    at tryOnImmediate (timers.js:534:15)
    at processImmediate [as _immediateCallback] (timers.js:514:5)
```

**Comment by meongx** [comment]
i have added pull request to fix this issue

in the meantime before it's merged, you can install my fork by doing
```
npm install -g meongx/node-huaban
```

**Comment by magicdawn** [comment]
fixed in version v1.4.0
